  public void testPhSuggestions2() throws Exception {
    doTest("ph2");
  }

